js_modules/dagit/packages/core/src/overview/OverviewSchedulesRoot.tsx

```diff
@@ -220,7 +220,9 @@ const UnloadableSchedulesAlert: React.FC<{
 };
 
 const UnloadableScheduleDialog: React.FC = () => {
-  const {data} = useQuery(UNLOADABLE_SCHEDULES_QUERY);
+  const {data} = useQuery(UNLOADABLE_SCHEDULES_QUERY, {
+    fetchPolicy: 'cache-first', // Is this correct?
```

Comment:
Not sure about this. Seems like it should requery in case unloadables change, e.g. if a repo is removed.